modules/ROOT/pages/servicenow/servicenow-reference-6.6.0.adoc

```diff
@@ -0,0 +1,413 @@
+= ServiceNow Connector Reference - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+ServiceNow Connector Version 6.6
+
+Anypoint Connector for ServiceNow (ServiceNow Connector) provides connections between Mule runtime engine (Mule) and ServiceNow apps.
+
+Release Notes: xref:release-notes::connector/servicenow-connector-release-notes-mule-4.adoc[ServiceNow Connector Release Notes]
+
+
+== Configurations
+
+[[config]]
+=== Config
+
```

Comment:
Is this because of the template, that the Configurations subtopic leads straight to Level-4 text?